jsonb/pom.xml

```diff
@@ -59,6 +59,10 @@
             <groupId>net.minidev</groupId>
             <artifactId>accessors-smart</artifactId>
         </exclusion>
+        <exclusion>
```

Comment:
Rather than exclude guava, can we just manually manage the version, as we're doing elsewhere?